config/plugins/urls.yml.example

```diff
@@ -16,44 +16,37 @@ blacklist: []  # List of patterns to match against URLs; if matched then the URL
 # - '.*\/\/vimeo\.com\/.*\/staffpicks\/[^\/]'  # Vimeo staff picks
 # - '.*\/\/bad-site\.com\/.{5,}\/.*'  # Dirs that are 5 chars long have massive titles
```

Comment:
It should be made clear that the blacklist only applies to the default handler. I feel that it should really apply to them all, but as it stands, this isn't clear.